solidity_contracts/src/ERC721.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.0;
+
+import {ERC721} from "solmate/tokens/ERC721.sol";
+
+contract KethToken is ERC721 {
```

Comment:
Not used ?

Reply:
indeed, I wanted to add it in the test block but forget after all trials and error for ERC20